### PR TITLE
Create cms-vm-2011.xml

### PR DESCRIPTION
**Pull request by AnxhelaDani**
WIP

### CODE REVIEW

**Comment by AnxhelaDani** [comment]
WIP- CMS VM 2011 and its contextualisation script (#815) 
[CMS_VM2011.txt](https://github.com/cernopendata/opendata.cern.ch/files/47954/CMS_VM2011.txt)

File: invenio_opendata/testsuite/data/cms/cms-vm.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-vm.xml]: There is only one empty line change in this PR?

**Comment by pherterich** [comment]
It's WIP, we have to move the content from the txt file and double check for some content with Kati.

File: invenio_opendata/testsuite/data/cms/cms-vm.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2011</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">The virtual machine image is based on CernVM (cernvm.cern.ch), with Scientific Linux CERN 6 (slc6), which is the environment needed for the CMSSW release 
+      to be used with the 2011 CMS open data. Access to the data is through XRootD.</subfield>
```
Comment (verdurin) [inline invenio_opendata/testsuite/data/cms/cms-vm.xml]: Oughtn't we to expand this point about access to the data, given the efforts to host the data on CVMFS?

File: invenio_opendata/testsuite/data/cms/cms-vm.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      <subfield code="a">The scripts provided create the CMS virtual machine image based on CernVM (cernvm.cern.ch), with Scientific Linux CERN 6 (slc6).</subfield>
+    </datafield>
+    <datafield tag="538" ind1=" " ind2=" ">
+      <subfield code="a">Use this with a Micro-CernVM available at</subfield>
+      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+    </datafield>
+    <datafield tag="581" ind1=" " ind2=" ">
+      <subfield code="a"><p> First, you need to run a normal Micro-CernVM, using your own context so you can login. Copy the attached files to the running VM and then 
+      login.<p>Inside the VM, you first need a copy of the latest Micro-CernVM release as a template for the Open Data image. Download it, inside the running VM, from: 
+      http://cernvm.cern.ch/releases/ucernvm-images.1.18-13.cernvm.x86_64/ucernvm-slc5.1.18-13.cernvm.x86_64.hdd <br/>placing it in the same directory as the other files.<br/>Run
```
Comment (verdurin) [inline invenio_opendata/testsuite/data/cms/cms-vm.xml]: The base image currently in use is at:

[http://cernvm.cern.ch/releases/testing/ucernvm-prod.2.5-2.cernvm.x86_64.hdd](url)

**Comment by tiborsimko** [comment]
Thanks, squashed together, fixed CDATA, renamed the file to stay close to 2010, and merged as:

- 4158bb897ce0f4e59046ce02f8b36171c233b448 CMS: VM image 2011 file name
- ba2b962bfd7033951d0c835ae4489c38b38b7f26 CMS: fix CDATA in record 251
- acfb86f0b0af4522443ec690572c115d2534a7a2 CMS: addition of `cms-vm.xml`

**Pull request closed without merging**